dev/CONTRIBUTING.md

```diff
@@ -100,87 +94,60 @@ Run `builder run check` before committing to ensure lint and tests are passing.
 
-### Component Style
-
-Victory is an ecosystem of components with similar language and methodologies. It should only require one learning curve, and avoid as much component specific domain knowledge as possible. Victory should expose an intuitive api, with sensible defaults that allow a user to render simple visualizations with little start up cost. Though Victory currently relies on d3, Victory components should not be simply React wrappers around d3.
-
-Victory components should:
-
-- Be created from the [FormidableLabs component generator](https://github.com/FormidableLabs/generator-formidable-react-component)
-- Be very well documented with [ecology](https://github.com/FormidableLabs/ecology)
-- Be named consistently _i.e._ `victory-bar`
-- Be React 0.14 compatible
-- *NOT* let 3rd party libraries like d3 interact with the DOM
-- Have sensible default props / fallback behaviors so that _something_ is rendered even when no props are provided
-- Produce sensible results if _only_ data is provided _i.e_ set the domain based on whatever data was provided
-- Have attractive default styling
-- Take a `style` prop, and intelligently merge provided (and scoped) styles with default styles
-
-```js
-// default styles
-const styles = {
-  parent: {
-    width: 500,
-    height: 300,
-    margin: 50
-  },
-  data: {
-    fill: "#756f6a",
-    opacity: 1,
-    stroke: "transparent",
-    strokeWidth: 0
-  },
-  labels: {
-    stroke: "none",
-    fill: "black",
-    fontFamily: "Helvetica",
-    fontSize: 10,
-    textAnchor: "middle"
-  }
-};
+## Victory Component Style Guide
 
-// merging styles:
-getStyles(props) {
-  if (!props.style) {
-    return styles;
-  }
-  const {data, labels, parent} = props.style;
-  return {
-    parent: _.merge({}, styles.parent, parent),
-    labels: _.merge({}, styles.labels, labels),
-    data: _.merge({}, styles.data, data)
-  };
-}
-
-// example style prop
+Victory is an ecosystem of modular components with similar language and methodologies. Because these components are designed to work together, a consistent set of patterns and conventions is necessary. Please read the following set of standards and examples if you are interested in writing a larger feature, or contributing a new victory component.
+
+### Flexibility
+
+Victory is designed to be as flexible as possible. Code should be free of hard-coded values or one-off solutions. The following patterns also help keep the library flexible: 
+
+**Seperating Rendered Components**
+
+Any rendered component (_i.e._ `<line/>`, `<text/>` and even `<g/>`) should be written as seperate components, and included via `defaultProps` so that they can be overridden the users. This pattern also allows us to use the same code base to support a react native version of Victory. 
```

Comment:
I'm a little confused by the mention of `<text/>` and `<g/>`, as they are SVG, not Victory. The note below to check https://github.com/FormidableLabs/victory-core/tree/master/src/victory-primitives and look at `<Point/>` _is_ really helpful thought